src/main/java/com/minecolonies/api/colony/jobs/IJob.java

```diff
@@ -284,13 +284,23 @@ default void triggerActivityChangeAction(boolean newState)
     /**
      * Whether the job is a guard
      *
-     * @return
+     * @return true if so.
      */
     default boolean isGuard()
     {
         return false;
     }
 
+    /**
+     * Whether the job is a guard that actively engages in combat
+     *
+     * @return true if so.
+     */
+    default boolean isCombatGuard()
```

Comment:
That is "non druid" ?

Reply:
Pretty much, I use this on the view and the guard assignment to check if the given guard is "any combat guard". Indeed to prevent hiring druids into the expedition.